share/spack/gitlab/cloud_pipelines/stacks/radiuss/spack.yaml

```diff
@@ -0,0 +1,131 @@
+# This file is currently being customized for quartz/toss_3_x86_64_ib.
+
+spack:
+  concretization: together   # separately
```

Comment:
Looking at the errors after you switched concretizers and used `gcc@7.5.0`, I recall that until #22887 is merged (which makes `spack ci rebuild` use the `--no-add` argument), these pipelines cannot be run on environments concretized `together`.  😬  I'm hoping that work is merged tonight, so you can either wait until it is, or else concretize `separately` until then.  But here you have `separately` commented out, does that mean that's what you'd prefer?

Reply:
Changed to concretize separately.